### ISSUE TITLE
Correct spellings incorrectly flagged by typos in CI

### ISSUE THREAD

**Issue by bobhy** [nushell/nushell#9196]
### Describe the bug

Typos in CI is opinionated about spelling, as it should be.  But what if its opinion is wrong?  PRs just fail, unit tests and other CI tests are not run and all progress on Nushell grinds to a halt.

Here's an example of the term "mis-highlighting" in general use: https://github.com/gskinner/regexr/issues/269

### How to reproduce

```
./typos
jq: jq-1.6
$ typos .
Warning: "mis" should be "miss".
error: `mis` should be `miss`
  --> ./crates/nu-command/src/formats/to/parsable.rs:78:103
   |
78 |                 description: "Without this help, simple types could produce unparsable text and cause mis-highlighting.",
   |           
```

This is from https://github.com/nushell/nushell/actions/runs/4973131732/jobs/8898771155?pr=9170, though maybe resolved by the time you read this.

### Expected behavior

1. I'd like a way to suppress a warning from typo, like we do with clippy when we disagree with its ruling, or are willing to live with the consequences in the codebase.
2. I'd like to have the typo checking tool integrated into `toolkit.nu` so I could catch these problems sooner, and maybe reword the offending text.

### Configuration

| key                | value                                    |
| ------------------ | ---------------------------------------- |
| version            | 0.79.1                                   |
| branch             | fix_9027                                 |
| commit_hash        | e5ed3249896c9864832c49b33979b44653b53426 |
| build_os           | linux-x86_64                             |
| build_target       | x86_64-unknown-linux-gnu                 |
| rust_version       | rustc 1.67.1 (d5a82bbd2 2023-02-07)      |
| rust_channel       | 1.67.1-x86_64-unknown-linux-gnu          |
| cargo_version      | cargo 1.67.1 (8ecd4f20a 2023-01-10)      |
| build_time         | 2023-05-14 11:00:40 -04:00               |
| build_rust_channel | debug                                    |
| features           | default, zip                             |
| installed_plugins  |                                          |

**Comment by fdncred** [comment]
I thought you could add words to it's dictionary but I haven't looked in a while.

**Comment by amtoine** [comment]
i think this is not a bug, just a missing feature :yum:

**Comment by bobhy** [comment]
The nub of the problem is that the typo checking crate isn't checked into
the source tree (is it?).  So even if I fix it locally, CI on github will
still fail fast.

**Comment by fdncred** [comment]
i've seen several ci's where part of the ci is to install a spellcheck package and then immediately after it's installed, add some known words to the dictionary. i'm not sure if that's possible with ours, but if it is, that's one solution.

**Comment by bobhy** [comment]
I think this issue is resolved by a resolution of #9924.

**Comment by fdncred** [comment]
I don't see a #9924. What am I missing?

**Comment by bobhy** [comment]
Oops, #9224

**Comment by amtoine** [comment]
> I think this issue is resolved by a resolution of #9924.
>
what do you mean? :thinking: 
there is a PR which should close both
- #9224 
- #9196
?

i do not see any PR mentionned in any of the two issues :open_mouth:

**Comment by AucaCoyan** [comment]
This was added back in #7892 . I realized that when I browsed the `.github` folder just _after_ modifying `toolkit.nu` 🤣 
I guess this closes this issue and #9196 too?

**Comment by fdncred** [comment]
sounds good. thanks!

**Comment by bobhy** [comment]
Agree, adding typos crate means I can now configure acceptable spellings and get CI to run if it was complaining about spelling.
But I think `toolkit.nu` should run typos as well, so I won't push a PR with typos in the first place.
